scripts/example.coffee

```diff
@@ -114,74 +174,71 @@ module.exports = (robot) ->
   robot.respond /wake me up when september ends/i, (res) ->
     res.reply 'Deployyyyyyy....... to production :D'
 
-  robot.respond /ankit/i, (res) ->
-    checkouts = robot.brain.get('features')
-    result = ':realtor: CORE WEB FEATURE BOXES :realtor: \n\n' ;
-    for own feature, name of checkouts
-      result = result.concat("> *#{name}* checked out #{feature}\n");
-    result = result.concat("\n\n> Any *feature* not listed is free for the taking! :parrotcop: ")
-    res.send result 
+  robot.respond /migrate features/, (res) ->
+    res.send "Migrating to the new DB format (key `featuresv2`)..."
+    (new Checkouts(robot)).migrate(res)
+    res.send "Completed migration :tayne:"
 
   robot.respond /who checked out (.*)\??/i, (res) ->
     feature = res.match[1]
-    checkouts = robot.brain.get("features") || {}
-    name = checkouts[feature] || 'nobody'
-
-    res.send "#{name} checked out #{feature}"
+    res.send "#{new Checkouts(robot).who(feature)} checked out #{feature}"
 
   robot.respond /show checkouts/i, (res) ->
```

Comment:
should we still keep our legacy checkouts until we don't migrate and test commands?

```coffee
robot.respond /show checkouts/i, (res) ->
    checkouts = robot.brain.get('features')
    result = ':realtor: CORE WEB FEATURE BOXES :realtor: \n\n' ;
    for own feature, name of checkouts
      result = result.concat("> *#{name}* checked out #{feature}\n");
    result = result.concat("\n\n> Any *feature* not listed is free for the taking! :parrotcop: ")
    res.send result 
```

and have 

```coffee
robot.respond /show checkouts_v2/i, (res) ->
    result = ':realtor: CORE WEB FEATURE BOXES :realtor: \n\n'
    checkouts = new Checkouts(robot)

    checkouts.each (feature, meta) =>
      since = (new ReadableTimeDiff(meta['when'])).toString()
      result += "> *#{meta['who']}* checked out *#{feature}* for #{since}\n"

    result += "\n\n> Any *feature* not listed is free for the taking! :parrotcop: "
    res.send result
```

This way we can test it live  and once we are comfortable will just swap v2 to main ? wdyt?